app/views/attach-pvc.html

```diff
@@ -23,7 +23,7 @@ <h2 class="text-center">No persistent volume claims.</h2>
               To claim storage from a persistent volume, refer to the documentation on <a target="_blank" ng-href="{{'persistent_volumes' | helpLink}}">using persistent volumes</a>.
             </p>
```

Comment:
Should we center this paragraph too for consistency with other empty state messages?